backend/src/main/java/com/dhbw/tutorsystem/tutorialRequest/CreateTutorialRequestRequest.java

```diff
@@ -0,0 +1,25 @@
+package com.dhbw.tutorsystem.tutorialRequest;
+
+import javax.validation.constraints.NotBlank;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@NoArgsConstructor
+public class CreateTutorialRequestRequest {
```

Comment:
Eigentlich sollte man bei einer Anfrage auch angeben können in welcher Studienrichtung man ist, oder? Also den `SpecialisationCourse` angeben (auch auf UI, zB mit Dropdown)

Reply:
Die Wahl des SpecialisationCourse im Frontend wurde verworfen damit ist das resolved